developerGuide.t2t

```diff
@@ -649,20 +649,26 @@ Generally, it has to be explicitly enabled even in archivers that do support it.
 ++ Manifest Files ++
 Each add-on package must contain a manifest file named manifest.ini.
 This must be a UTF-8 encoded text file.
-This manifest file contains key = value pares declaring info such as the add-on's name, version and description.
+This manifest file contains key = value pairs declaring info such as the add-on's name, version and description.
 
 +++ Available Fields +++
 Although it is highly suggested that manifests contain all fields, the fields marked as mandatory must be included.
-Otherwise, the add-on will not install.  
+Otherwise, the add-on will not install.
+
 - name: A short unique name for the add-on. This is used to differentiate add-ons internally and is not shown to the user. (Mandatory)
 - summary: The name of the add-on as shown to the user. (Mandatory)
 - version: The version of this add-on; e.g. 2.0. (Mandatory)
 - author: The author of this add-on, preferably in the form Full Name <email address>; e.g. Michael Curran <mick@kulgan.net>. (Mandatory)
 - description: A sentence or two describing what the add-on does.
 - url: A URL where this add-on, further info and upgrades can be found.
 - docFileName: The name of the main documentation file for this add-on; e.g. readme.html. See the [Add-on Documentation #AddonDoc] section for more details.
+- minimumNVDAVersion: The minimum required version of NVDA for this add-on to be installed or enabled.
+- lastTestedNVDAVersion: The last version of NVDA this add-on has been tested with.
 -
 
+The minimumNVDAVersion and lastTestedNVDAVersion keys are currently not mandatory.
+However, it is highly recommended that these keys be included as they are expected to become mandatory keys in the future.

```

Comment:
Although the user *can* install an add-on without these keys existing in the manifest, surely from an add-on author's point of view, from now these *are* mandatory?